src/libmongoc/examples/example-create-indexes.c

```diff
@@ -1,92 +1,106 @@
-/* gcc example-create-indexes.c -o example-create-indexes $(pkg-config --cflags
- * --libs libmongoc-1.0) */
-
-/* ./example-create-indexes [CONNECTION_STRING [COLLECTION_NAME]] */
+// example-create-indexes creates, lists and deletes an index from the
+// `test.test` collection.
 
 #include <mongoc/mongoc.h>
-#include <stdio.h>
-#include <stdlib.h>
+#include <stdlib.h> // abort
+
+#define FAIL(...)                                        \
+   if (1) {                                              \
+      fprintf (stderr, "failed on line %d\n", __LINE__); \
```

Comment:
```suggestion
      fprintf (stderr, "Failure at %s:%d\n", __FILE__, __LINE__); \
```

just because tools often recognize `file:line` and make it a clickable link.